### DISCUSSION TITLE
Simple example returns unexpected <head> tag

### DISCUSSION THREAD

**Discussion by guillaumegarcia13** [unifiedjs/unified#130] [Q&A] [answered]
I can't get what I'm doing wrong here with this HTML snippet converted to AST then back to HTML
```
        var unified  = require('unified');
        var parser   = require('rehype-parse');      // parser (converts HTML to Abstract Syntax Tree)
        var compiler = require('rehype-stringify');  // compiler (converts AST to HTML)

        var htmlContent = `<a href="https://alpha.com" class="bravo" download></a>`;

        var tree = unified()
            .use(parser)
            .parse(htmlContent);

        var output = unified()
            .use(compiler, {
                quoteSmart      : true,
                closeSelfClosing: true,
                omitOptionalTags: true,
                entities: { useShortestReferences: true }
            })
            .stringify(tree);
        
        console.log(output);
```

In my terminal, I get:
`<head><a href="https://alpha.com" class="bravo" download></a>`

it seems that empty <head></head> made it (unexpectedly, at least to me) to the output.

Is this because of the options used with the compiler?
Am I missing something?

**Comment by wooorm** [comment, marked as answer]
You're missing the fragment option: https://github.com/rehypejs/rehype/tree/main/packages/rehype-parse#optionsfragment

rehype-parse expects a whole document by default.
